### ISSUE TITLE
Can't freeze model Data loss: not an sstable (bad magic number)

### ISSUE THREAD

**Issue by zhangchunsheng** [tensorflow/models#2837]
Training with my own data, Get this when I train model. More info here: https://github.com/zhangchunsheng/tensorflow-rbnr
config: faster_rcnn_numobj.config
tensorflow:1.4.0, models project is newest.
gen.py will make the record file
I train it on my Mac.

INFO:tensorflow:Summary name /clone_loss is illegal; using clone_loss instead.
/Library/Python/2.7/site-packages/tensorflow/python/ops/gradients_impl.py:96: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
"Converting sparse IndexedSlices to a dense Tensor of unknown shape. "

then can't freeze model

python object_detection/export_inference_graph.py --input_type image_tensor --pipeline_config_path models/faster_rcnn_numobj.config --trained_checkpoint_prefix models/train/model.ckpt-* --output_directory models/result/

2017-11-19 22:26:38.039878: W tensorflow/core/util/tensor_slice_reader.cc:95] Could not open train/model.ckpt-0.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

**Comment by onlyNata** [comment]
I have the same problem.Update TF-1.4 and the model(git pull ..) to the latest, then ,is OK~
